src/__tests__/constants.ts

```diff
@@ -1,4 +1,14 @@
-import { Profile, ProfileActions, Session, ShowData } from '../types';
+import {
+    Profile,
+    ProfileActions,
+    Session,
+    ShowData,
+    SeasonDetails,
+    Episode,
+    EpisodeDetails,
+} from '../types';
+
+export const TMDB_BASE_PATH = 'https://image.tmdb.org/t/p/w500';
```

Comment:
I wonder if we should change the name of this to `TMDB_IMG_BASE_PATH` or something like that. Since it is not the API base path.